src/core/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -70,18 +70,29 @@
                     <TextBlock Margin="12,0,0,0" x:Uid="FancyZones_LaunchEditorButtonControl"/>
                 </StackPanel>
             </Button>
-            
-            <TextBlock x:Uid="FancyZones_ZoneBehavior_GroupSettings"
+
+            <StackPanel Orientation="Horizontal">
+                <TextBlock x:Uid="FancyZones_ZoneBehavior_GroupSettings"
                        Style="{StaticResource SettingsGroupTitleStyle}"
                        />
 
+                <Viewbox Height="12" Width="12" Margin="5, 30, 0, 0">
+                    <PathIcon
```

Comment:
PathIcons will use the default Foreground, right? So it works fine in Dark Theme as well.

Reply:
yes, it adheres to the foreground color.